modules/android/examples/codesnippet_collection.java

```diff
@@ -2829,13 +2847,12 @@ public void replicationCustomHeaderExample(
 
     public void replicationPushFilterExample(Set<Collection> collections) throws URISyntaxException {
         // tag::replication-push-filter[]
-        CollectionConfiguration collectionConfig = new CollectionConfiguration()
-            .setPushFilter((document, flags) -> flags.contains(DocumentFlag.DELETED)); // <1>
+        Set<CollectionConfiguration> collectionConfigs = CollectionConfiguration.fromCollections(collections);
```

Comment:
The above uses collConfigs, so I think it would be better to use the same name, whether this one of the other one. Please check the other code snippets as well.